sklearn/metrics/ranking.py

```diff
@@ -294,6 +294,12 @@ def _binary_clf_curve(y_true, y_score, pos_label=None, sample_weight=None):
     thresholds : array, shape = [n_thresholds]
         Decreasing score values.
     """
+    # Check to make sure y_true is valid
+    y_type = type_of_target(y_true)
+    if not (y_type == "binary" or
+            (y_type == "multiclass" and pos_label is not None)):
+        raise ValueError("{0} format is not supported".format(y_type))
```

Comment:
@lesteve, my choice of error message was copy pasted from other portions of this code.  I chose the language to be consistent with the other instances of similar errors in `ranking.py`.